java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
+	http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
+  <http:listener-config name="HTTP_listener_config" >
+  <http:listener-connection host="0.0.0.0" port="8081" />  
+  </http:listener-config>
+  <flow name="name1">
+        <http:listener path="/calculatePath" config-ref="HTTP_listener_config" doc:name="Listener" doc:id="tlryte" allowedMethods="POST"/>
+        <java:new class="com.me.TaxCalculator" constructor="TaxCalculator()" doc:name="New" doc:id="ucwbpq" target="taxCalculator"/>
+</flow>
+</mule>
+----
+
+=== Add an Invoke 
+
+To call your method, add an *Invoke* operation:
+
+. Click the *Invoke* component on the canvas.
+. In the *General* tab:
+.. Enter `#[vars.taxCalculator]` as the *Instance* on which the method is invoked.
+.. Select the *Class* that contains the method to use, in this case it is `com.me.TaxCalculator`.
+.. Select the *Method* to use: `calculateTax(java.lang.Double,java.lang.Integer)`.
++
+The list of methods shown are standard non-static Java methods in addition to the methods you created. 
+. In the *Advanced* tab, assign the Invoke operation to a target variable called `taxResult`.
+. In the XML editor, add the Java arguments as a DataWeave expression:
++
+[source,xml,linenums]
+----
+<java:args>#[{
+        arg0: payload:price,
+        arg1: payload:percentAdditionalTax
+      }]</java:args>
+----
+
+Your XML code looks similar to this now:
+
+[source,xml,linenums]
+----
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:java="http://www.mulesoft.org/schema/mule/java" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
+  <http:listener-config name="HTTP_listener_config">
+    <http:listener-connection host="0.0.0.0" port="8081" />  
+  </http:listener-config>
+  
+  <flow name="name1">
+    <http:listener path="/calculatePath" config-ref="HTTP_listener_config" doc:name="Listener" doc:id="tlryte" allowedMethods="POST"/>
+    <java:new class="com.me.TaxCalculator" constructor="TaxCalculator()" doc:name="New" doc:id="ucwbpq" target="taxCalculator"/>
+    <java:invoke instance="#[vars.taxCalculator]" class="com.me.TaxCalculator" method="calculateTax(java.lang.Double,java.lang.Integer)" doc:name="Invoke" doc:id="ulrzkz" target="taxResult"/>
+    <java:args>#[{
+        arg0: payload:price,
+        arg1: payload:percentAdditionalTax
```

Comment:
I asked Josefina about something people are likely to want to do, which is change arg0 and arg1 to meaningful parameter names. Not enough time, but something to keep in mind.